.github/workflows/build.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [8,11]
+        java: [8,11,14,17]
```

Comment:
I would prefer to keep it `[8,11,17]` as it's easier to use those for local development with sdkman.